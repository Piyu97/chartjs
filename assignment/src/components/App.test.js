import React from "react"
import App from "../App"
import renderer from "react-test-renderer"

test("Snapshot", () => {
    const component = renderer.create( 
        <App/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})