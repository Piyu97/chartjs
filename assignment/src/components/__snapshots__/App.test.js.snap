// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div>
  <h3
    className="p-3 text-center bg-dark text-warning "
  >
    Assignment on Charts
  </h3>
  <div
    className="table-responsive mt-5"
  >
    <table
      className="table table-bordered"
    >
      <thead
        className="bg-dark text-white"
      >
        <tr>
          <td>
            SI.NO
          </td>
          <td>
            Email
          </td>
          <td>
            First Name
          </td>
          <td>
            Last Name
          </td>
          <td>
            Created Date
          </td>
          <td>
            Orders
          </td>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
  <ul
    className="float-right"
  >
    <li
      className="page-item list-inline-item border border-danger p-2"
      onClick={[Function]}
    >
      Previous
    </li>
    <li
      className="page-item list-inline-item border border-success p-2"
    >
      1
    </li>
    <li
      className="page-item list-inline-item border border-danger p-2"
      onClick={[Function]}
    >
      Next
    </li>
  </ul>
</div>
`;
